Form-Controls/styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+    background-color: blue;
+  }
+  
+  header {
+    text-align: center;
+    color: white;
+  }
+
+
+  form {
+    padding: 20px;
+    border-radius: 10px;
+    background-color: white;
+    margin: 0 auto;
+    width: 50%;
+  }
+  
+  label {
+    font-size: 18px;
+    margin-top: 10px;
+  }
+  
+  input[type="text"], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+  
+  input[type="submit"] {
+    width: 100%;
+    background-color: orange;
+    color: white;
+    padding: 14px 20px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+}
+input[type="submit"]:hover {
+    background-color: green;
```

Comment:
What would happen if a user was accessing this without a mouse?

https://css-tricks.com/keyboard-only-focus-styles/